### ISSUE TITLE
Support running multiple scripts, both in parallel or sequential

### ISSUE THREAD

**Issue by macarie** [egoist/dum#18]
I don't know if this is out-of-scope for dum, but I find that it could be a perfect (and faster) alternative for [npm-run-all](https://github.com/mysticatea/npm-run-all) (that, at this point, I don't even think it's maintained anymore).

I'd love it if dum could run multiple scripts in one command, e.g. `dum clean build`, and, with a flag, run them in parallel, so something like `dum --parallel build:ts build:dts`.

Another impressive feature of npm-run-all is its *glob-like* script-name resolution (e.g. `build:*`). It would be *chef's kiss* if this could be implemented and translated to "run every script that matches this pattern in parallel." This would make it very easy to have one script, `dum clean build:*`, that `clean`s the output directory, and, **after that**, runs both `build:ts` and `build:dts` in parallel.

**Comment by egoist** [comment]
I would love to implement this feature, probably through a syntax like `dum  "foo bar"`, without quotes `bar` will be passed to the `foo` script as cli args.
